cvat/apps/dataset_manager/annotation.py

```diff
@@ -72,11 +72,23 @@ def has_overlap(a, b):
             # a <= b
             return 0 <= min(b, stop) - max(a, start)
 
+        elements = track.get('elements', [])
+        if elements:
+            # skeletons usually have only one shape defined on initial frame
+            # anyway, for them we decided if they are inside range based on child elements
+            return any(AnnotationIR._is_track_inside(t, start, stop) for t in elements)
+
         prev_shape = None
         for shape in track['shapes']:
+            if shape['frame'] == start and shape['outside']:
+                # corner case when the only shape on segment is outside frame on start frame
+                prev_shape = shape
+                continue
```

Comment:
I'm not totally sure this function should be responsible for deciding anything about segments. Technically, there could have been the stop frame, if it wasn't dropped in the `slice_track` function. Thus, there is another possible place to fix the problem - in [this line](https://github.com/cvat-ai/cvat/blob/5bb83a150fbdfecb0c46b5b0a314ed6c98215e5e/cvat/apps/dataset_manager/annotation.py#L159) (I can guess that that check also was introduced because of this problem).